### ISSUE TITLE
fix: Setting button disabled should also set aria-disabled

### ISSUE THREAD

**Issue by m-akinc** [microsoft/fast#6783]
# 🐛 Bug Report

Setting the `disabled` attribute on the button should also set `aria-disabled` on that element.

## 💻 Repro or Code Sample

Set `disabled` on a FAST button and see that `aria-disabled` is not true.

## 🤔 Expected Behavior

See title.

## 😯 Current Behavior

Button's `aria-disabled` is not true.

## 💁 Possible Solution

Add `<template aria-disabled=${x => x.disabled}>` to the button template.

## 🔦 Context

This is causing problems for some [Playwright](https://github.com/microsoft/playwright) tests we are trying to write. The Playwright `toBeDisabled()` assertion [will only succeed if native form elements have the `disabled` attribute or non-native custom elements have `aria-disabled=true`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-disabled). This means our FAST-based button with the `disabled` attribute is currently failing that assertion.

**Comment by chrisdholt** [comment]
I don't think this would be valid or expected - the actual element receiving focus is the button within the shadow root. The host is considered generic, there is no role or anything which would necessitate `aria-disabled` and adding said role would likely trigger a new violation as the ARIA attribute is now present when not necessary. This sounds like a function of the test and how it's written. Here's our playwright test to validate the disabled state (first test): https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/button/button.pw.spec.ts
